quantum/process_keycode/process_tap_dance.h

```diff
@@ -39,15 +39,12 @@ typedef struct
 
 #define ACTION_TAP_DANCE_DOUBLE(kc1, kc2) { \
     .fn = { NULL, qk_tap_dance_pair_finished, qk_tap_dance_pair_reset }, \
-    .user_data = (void *)&((qk_tap_dance_pair_t) { kc1, kc2 })  \
+    .user_data = (void *)&((qk_tap_dance_pair_t) { kc1, kc2 }),  \
   }
 
-#define ACTION_TAP_DANCE_FN(user_fn) {  \
-    .fn = { NULL, user_fn, NULL } \
-  }
-
```

Comment:
Please don't remove `ACTION_TAP_DANCE_FN`, it is a useful macro to simplify cases that only need an `on_finished` callback (I'm using it in my keymap, and while using `ACTION_TAP_DANCE_FN_ADVANCED` would be similarly easy, there is no harm in having a simpler version too.)